client/src/views/files/FoldersPage.vue

```diff
@@ -267,6 +288,50 @@ const msSorterLabels = {
   desc: 'FoldersPage.sort.desc',
 };
 
+const tabBarActions = computed(() => {
+  const selectedEntries = getSelectedEntries();
+  const isReadOnly: boolean = workspaceInfo.value ? workspaceInfo.value.currentSelfRole === WorkspaceRole.Reader : true;
+  const actions: MenuAction[] = [];
+  if (!isReadOnly) {
+    if (selectedEntries.length === 1) {
+      actions.push({ label: 'FoldersPage.tabbar.rename', action: async () => await renameEntries(getSelectedEntries()), icon: create });
+    } else {
+      actions.push({ label: 'FoldersPage.tabbar.duplicate', action: async () => await copyEntries(getSelectedEntries()), icon: duplicate });
+    }
+    actions.push({ label: 'FoldersPage.tabbar.move', action: async () => await moveEntriesTo(getSelectedEntries()), icon: arrowRedo });
+    actions.push({
+      label: 'FoldersPage.tabbar.delete',
+      action: async () => await deleteEntries(getSelectedEntries()),
+      icon: trashBin,
+      danger: true,
+    });
+  }
+  if (selectedEntries.length === 1 && selectedEntries[0].isFile()) {
+    actions.push({ label: 'FoldersPage.tabbar.open', action: async () => await openEntries(getSelectedEntries()), icon: open });
+  }
+  if (selectedEntries.length > folders.value.getSelectedEntries().length) {
+    actions.push({ label: 'FoldersPage.tabbar.download', action: async () => await downloadEntries(getSelectedEntries()), icon: download });
+  }
+  if (selectedEntries.length === 1) {
+    if (isReadOnly) {
+      actions.push(
+        { label: 'FoldersPage.tabbar.copyLink', action: async () => await copyLink(getSelectedEntries()), icon: link },
+        { label: 'FoldersPage.tabbar.share', action: async () => await shareEntries(), icon: shareSocial },
+        { label: 'FoldersPage.tabbar.details', action: async () => await showDetails(getSelectedEntries()), icon: informationCircle },
+      );
+    } else {
+      actions.push(
+        { label: 'FoldersPage.tabbar.duplicate', action: async () => await copyEntries(getSelectedEntries()), icon: duplicate },
+        { label: 'FoldersPage.tabbar.copyLink', action: async () => await copyLink(getSelectedEntries()), icon: link },
+        { label: 'FoldersPage.tabbar.share', action: async () => await shareEntries(), icon: shareSocial },
```

Comment:
I just discovered that we had a `shareEntries` that copies the mountpoint path to the clipboard, as well as a `copy link` that copies the file link to the clipboard.

I think `shareEntries` should just be removed (and only appear in the details dialog), it's confusing. Especially with that name, and it can't be done on web anyway. Could you do that please?